app/panel/components/Settings/TrustAndRestrict.jsx

```diff
@@ -86,11 +86,9 @@ class TrustAndRestrict extends React.Component {
 	/**
 	 * Implement adding site to the list of whitelisted or blacklisted sites
 	 * This routine valides entered url and checks if the entered url is a duplicate, or
-	 * if it has been alreday added to the opposite list. Displays appropriate warnings.
+	 * if it has been already added to the opposite list. Displays appropriate warnings.
 	 */
 	addSite() {
-		// from node-validator
-		const isValidUrlRegex = /^(?!mailto:)(?:(?:https?|ftp):\/\/)?(?:\S+(?::\S*)?@)?(?:(?:(?:[1-9]\d?|1\d\d|2[01]\d|22[0-3])(?:\.(?:1?\d{1,2}|2[0-4]\d|25[0-5])){2}(?:\.(?:[1-9]\d?|1\d\d|2[0-4]\d|25[0-4]))|(?:(?:[a-z\u00a1-\uffff0-9]+-?)*[a-z\u00a1-\uffff0-9]+)(?:\.(?:[a-z\u00a1-\uffff0-9]+-?)*[a-z\u00a1-\uffff0-9]+)*(?:\.(?:[a-z\u00a1-\uffff]{2,})))|localhost)(?::\d{2,5})?(?:\/[^\s]*)?$/i;
```

Comment:
We still need front-end validation here so that we can prevent users from adding urls in unsupported formats. You should check that the url entered is either a valid url or a valid regex, and show an error if not.

Reply:
I checked if it's a valid URL, wildcard, or regex and updated the error messages. Let me know if you'd like to change anything else